packages/algob/src/internal/deployer.ts

```diff
@@ -242,8 +256,8 @@ class DeployerBasicMode {
    * Send signed transaction to network and wait for confirmation
    * @param rawTxns Signed Transaction(s)
    */
-  async sendAndWait (rawTxns: Uint8Array | Uint8Array[]): Promise<ConfirmedTxInfo> {
-    return await this.algoOp.sendAndWait(rawTxns);
+  sendAndWait (rawTxns: Uint8Array | Uint8Array[]): Promise<ConfirmedTxInfo> {
+    return this.algoOp.sendAndWait(rawTxns);
```

Comment:
Why do we remove `async/await` from here? I thought sending and waiting tx confirmation from network should be promise based? And note: you return `Promise<ConfirmedTxInfo>`

Reply:
Yes, this is what the return type is. 
we don't need async / await here because we don't need result of the function here, it can stay in prromise

Reply:
Or as you suggested below, we can change to `getDeployedApp`